dashboard/v1.0/src/App.tsx

```diff
@@ -1,20 +1,13 @@
 import React from 'react';
 import './App.css';
-import Header from './components/layouts/Header';
-import Sidebar from './components/layouts/Sidebar';
+import GlobalLayout from './components/layouts/GlobalLayout';
 import Navigator from './router/Navigation';
 
 function App() {
   return (
-    <div className="App">
-      <div className="sidebar">
-        <Sidebar />
-      </div>
-      <div className="inner-component">
-        <Header />
-        <Navigator />
-      </div>
-    </div>
+    <GlobalLayout>
+      <Navigator />
+    </GlobalLayout>
```

Comment:
Could you please tell why is this necessary?
Also, `GlobalLayout` should be changed to something more relatable like `Wrap` or something related to the topic. _(I am not good at naming)_

Reply:
It's a layout that uses composition. So `GlobalLayout` is perfect. We can use `BasicLayout` otherwise.

Reply:
I just want to avoid `Global` from naming. Why not give the name related to something you want to implement?

Reply:
`BasicLayout` or `DefaultLayout` will be good?

Reply:
`BasicLayout` or `DefaultLayout` will be good?

Reply:
`BasicLayout` or `DefaultLayout` will be good?

Reply:
cool